packages/f/fmtlog/xmake.lua

```diff
@@ -8,7 +8,7 @@ package("fmtlog")
     add_versions("v2.2.1", "9bc2f1ea37eece0f4807689962b529d2d4fa07654baef184f051319b4eac9304")
     add_versions("v2.1.2", "d286184e04c3c3286417873dd2feac524c53babc6cd60f10179aa5b10416ead7")
 
-    add_deps("cmake", "fmt")
+    add_deps("cmake", "fmt 10.2.1")
```

Comment:
isn't `fmt 10` enough? this allows for minor and patch version mismatch

Reply:
yes that works, thanks